tests/api.c

```diff
@@ -38825,6 +38828,21 @@ static void test_wolfSSL_CTX_ctrl(void)
     AssertIntEQ(wolfSSL_EC_KEY_generate_key(ecKey),1);
 #endif
 
+#ifdef HAVE_ED25519
```

Comment:
This test is in a `OPENSSL_ALL` section. If this should be available in a `OPENSSL_EXTRA` section then please move it into such a section. Either by moving to a different test or create a new test func.